axon-framework/modules.md

```diff
@@ -153,3 +155,69 @@ This module provides support for distributed tracing of Axon applications. The [
 ### Axon Tracing Spring Boot Starter
 
 This module provides Spring auto-configuration on top of the `axon-tracing` module
+
+## Axon Bill of Materials
+
+Next to the main framework modules and the extensions, Axon also has a [Bill of Materials](https://en.wikipedia.org/wiki/Software_bill_of_materials), or BOM. 
```

Comment:
```suggestion
Next to the main framework modules and the extensions, Axon also has a [Bill of Materials](https://en.wikipedia.org/wiki/Software_bill_of_materials), or BOM. 
```
"Next to" is not commonly used in English, unless you're referring to a physical location of something. Instead I would use "In addition to"

Reply:
Did not know that! :see_no_evil: Thanks pointing that out. Adjusted per recommendation.

Reply:
Did not know that! :see_no_evil: Thanks pointing that out. Adjusted per recommendation.